config/locales/en.yml

```diff
@@ -2174,4 +2174,18 @@ en:
     subheading_2: How to share results
     subsection_2: To share the results of a check, save the result declaration as a PDF. You can do this on the results page at the end of the check.
     cta: Start a new eligibility check
+  helps:
+    show:
+      title: Help
+      get_help: Get help to use this service
+      contact: "Contact our customer services team if you need help doing things online, or want support with legal aid administration or training:"
+      telephone: 0300 200 2020
+      webchat_html: on <a href="https://legalaidlearning.justice.gov.uk/totara/dashboard/index.php?id=34" target="_blank" rel="noreferrer noopener">our training website</a>
```

Comment:
Shouldn't this be:

`webchat_html: webchat&#58; on <a href="https://legalaidlearning.justice.gov.uk/totara/dashboard/index.php?id=34" target="_blank" rel="noreferrer noopener">our training website</a>`